### ISSUE TITLE
How to get 3d coordinates values?

### ISSUE THREAD

**Issue by abdulahad7** [ildoonet/tf-pose-estimation#663]

**Comment by insidedctm** [comment]
I'm pretty sure you can't with this model - it's 2D pose estimation

**Comment by ArimaValanImmanuel** [comment]
> I'm pretty sure you can't with this model - it's 2D pose estimation

any idea on how to write our own posenet for 3d estimation??

**Comment by insidedctm** [comment]
You could have a look at this https://github.com/DenisTome/Lifting-from-the-Deep-release. From the lab where I did my MSc but haven’t tried it myself.

**Comment by ArimaValanImmanuel** [comment]
> You could have a look at this https://github.com/DenisTome/Lifting-from-the-Deep-release. From the lab where I did my MSc but haven’t tried it myself.

Thank you sir

**Comment by abdulahad7** [comment]
I tried it, using 2d from this version and 3d from LFTD works fine or you can clone the old repository of this openpose which includes 3d estimation too.

**Comment by insidedctm** [comment]
> I tried it, using 2d from this version and 3d from LFTD works fine or you can clone the old repository of this openpose which includes 3d estimation too.

Interesting. which branch/commit has the 3D stuff in? Any idea why it’s not in the main branch?

**Comment by abdulahad7** [comment]
Yes it is [here](https://github.com/ildoonet/tf-pose-estimation/tree/6980660b6f50653646a33c5a493d4c51d4335a3f). Here is the [helping link](https://www.youtube.com/watch?v=imheSFUzNzE&list=RDCMUC2W0aQEPNpU6XrkFCYifRFQ&start_radio=1).

**Comment by insidedctm** [comment]
Lovely, thank you. It looks like the old version from this repo is using LFTD as well, is that right?
